### ISSUE TITLE
MessagePack deserialization

### ISSUE THREAD

**Issue by Isuru-Nanayakkara** [luckymarmot/Paw-Issues#47]
Can you please add deserialization for API responses returned in [MessagePack](http://msgpack.org/) format?

**Comment by mittsh** [comment]
That's a great idea! 😄  We will try to add this next.

**Comment by trinode** [comment]
I just found this on google after hoping for the same, It'd be good to use the JSON request build tools, convert to msgpack, make request, receive msgpack, convert to JSON -> display result

**Comment by mittsh** [comment]
Yep that's the idea @trinode 😄  We're currently planning these features…

**Comment by trinode** [comment]
Any updates on this? That would be really useful for us?
